Gems/Atom/Feature/Common/Assets/Passes/ChromaticAberration.pass

```diff
@@ -0,0 +1,63 @@
+{
+    "Type": "JsonSerialization",
+    "Version": 1,
+    "ClassName": "PassAsset",
+    "ClassData": {
+        "PassTemplate": {
+            "Name": "ChromaticAberrationTemplate",
+            "PassClass": "ChromaticAberrationPass",
+            "Slots": [
+                {
+                    "Name": "Input",
+                    "SlotType": "Input",
+                    "ScopeAttachmentUsage": "Shader"
+                },
+                {
+                    "Name": "Output",
+                    "SlotType": "Output",
+                    "ScopeAttachmentUsage": "Shader",
+                    "LoadStoreAction": {
+                        "LoadAction": "Clear"
+                    }
+                }
+            ],
+            "ImageAttachments": [
+                {
+                    "Name": "ChromaticAberration",
+                    "SizeSource": {
+                        "Source": {
+                            "Pass": "This",
+                            "Attachment": "Input"
+                        }
+                    },
+                    "FormatSource": {
+                        "Pass": "This",
+                        "Attachment": "Input"
+                    }
+                }
+            ],
+            "Connections": [
+                {
+                    "LocalSlot": "Output",
+                    "AttachmentRef": {
+                        "Pass": "This",
+                        "Attachment": "ChromaticAberration"
+                    }
+                }
+            ],
+            "PassData": {
+                "$type": "ComputePassData",
+                "ShaderAsset": {
+                    "FilePath": "Shaders/PostProcessing/ChromaticAberration.shader"
+                },
+                "Make Fullscreen Pass": true
+            },
+            "FallbackConnections": [
```

Comment:
In SsaoCompute and SubsurfaceScattering this is located after PassData. I will move it since you asked but can a reviewer comment on the proper place going forward? And perhaps raise an issue for whichever passes are incorrect? I think it is important to have consistency (unless there is a functional reason for swapping them in come cases?).